x-pack/plugins/siem/public/pages/case/components/use_push_to_service/index.tsx

```diff
@@ -149,11 +164,11 @@ export const usePushToService = ({
           <>{pushToServiceButton}</>
         ),
       pushCallouts:
-        errorsMsg.length > 0 ? (
+        errorsMsg.length > 0 || connectors.length === 0 ? (
```

Comment:
I am wondering why we need to add `|| connectors.length === 0` if we have should have an error if have `connectors.length === 0`

Reply:
yeah you're right... im not sure, i feel like i was seeing the push callout before loading so this is how i handled it... which is dumb, i should fix it up in the errors. ill try to reproduce and fix